### ISSUE TITLE
Can't compile under a Release scheme

### ISSUE THREAD

**Issue by marcshilling** [Kureev/react-native-blur#30]
I added RNBlur to my project correctly and everything, builds fine and is working correctly in my React Native app...with a Debug build that is.

If I try to do a Release build in Xcode, I get this error:
```
clang: error: no such file or directory: '/Users/marcshilling/Library/Developer/Xcode/DerivedData/Wego-fkagpfjtiozdwmhallyibbxulerw/Build/Products/Release-iphonesimulator/libRNBlur.a'
```

I've tried some obvious debugging...cleaning the project, comparing the xcodeproj file to other React Native libraries I'm using...what am I missing?

**Comment by marcshilling** [comment]
Found it! I had to go to the RNBlur target, and set 'Build Active Architecture Only' to 'No' for 'Release'. I can make a PR if you'd like me to.

**Comment by Kureev** [comment]
Yeah, nice catch! PR is very welcome!

**Comment by Kureev** [comment]
Released fix in `0.7.5`
